hypothesis-python/src/hypothesis/searchstrategy/types.py

```diff
@@ -146,7 +146,7 @@ def from_typing_type(thing):
     frozenset: st.builds(frozenset),
     dict: st.builds(dict),
     # Built-in types
-    type: st.sampled_from([type(None), bool, int, str, list, set, dict]),
+    type: st.sampled_from([type(None), bool, int, float, str, list, tuple, set, dict, complex, ]),
```

Comment:
This line is now too long - we'll need to reformat it, like:

```python
type: st.sampled_from([
    type(None), bool, int, float, str, list, tuple, set, dict, 
    complex,
]),
```

It would also be nice to add the `bytes` type, as mentioned in #1685.  A more difficult but complete option would be to delete the current entry for `type` and insert it based on the other contents, i.e.

```python
_global_type_lookup = {
   ...
}
_global_type_lookup[type] = st.sampled_from(
    sorted(_global_type_lookup.keys(), key=TODO_MAGIC)
)
```

where the `TODO_MAGIC` function sorts `NoneType` first, then other types in a reliable order (e.g. by name).  But formatting the line and adding `bytes` would be plenty!

Reply:
Thanks @Zac-HD! I will take a look of these things and make the changes.